.github/workflows/ci.yml

```diff
@@ -10,9 +10,10 @@ jobs:
   build_and_test_pinned:
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        python-version: ['3.7', '3.8', '3.9', '3.10']  # 'pypy-3.7'
+        python-version: ['3.8', '3.9', '3.10']  # 'pypy-3.7'
```

Comment:
It would be unfortunate to drop 3.7 (an active Python version) from testing, as well as change the package to require `trio >= 0.24.0`.

`trio.testing.RaisesGroup()` is just a utility, and there are ways to handle maybe-wrapped-in-exception-group in tests without adding the dependency.

Reply:
According to https://www.python.org/downloads/release/python-3717/, python 3.7 is EOL at this point.

The package also doesn't require `trio >= 0.24.0`, just the `dev` extra. Although it would be fairly easy to fix that yes.

Reply:
You're right, I missed both those points.

It would be nice to keep the ability to run tests with older trio versions.   (My org still runs trio 0.21.0)

Reply:
Pushed a new version that works with trio 0.21.0 on my machine